bindings/python/generate_tests.py

```diff
@@ -57,7 +57,7 @@ def generate_file(object_name, callables, properties):
 """
 import Hamlib
 '''
-    filename = f"test_{object_name.replace(".", "_")}_class.py"
```

Comment:
Your change looks good; the reason is that you are using Python 3.11 while I'm using 3.13 and my syntax is [allowed since 3.12](https://docs.python.org/3.12/whatsnew/3.12.html#pep-701-syntactic-formalization-of-f-strings)

Reply:
The m4 macros can be unintentionally updated in situations like issue #1746 documents.